### PR TITLE
Update redeem NFT modal

### PR DESCRIPTION
**Pull request by johnnyd-eth**
## What does this PR do and why?

- shows redeem value
- use same NFT cards as project page
- fixes pending and confirmed states of modals
- disable OK button when no NFTs selected

## **To test:**
1. Go to a Goerli NFT project locally (preview link doesn't load NFTs)
2. Buy some NFTs
3. Redeem them
4. Check redeem was successful:
- your NFT count reduced on reload
- your wallet ETH balance has increased

https://user-images.githubusercontent.com/96150256/207519198-483c46ea-e01c-4b20-bf8f-b52a1768249f.mp4

## Acceptance checklist

- [ ] I have evaluated the [Approval Guidelines](https://github.com/jbx-protocol/juice-interface/blob/main/CONTRIBUTING.md#approval-guidelines) for this PR.
- [ ] I have tested this PR in [all supported browsers](https://github.com/jbx-protocol/juice-interface/blob/main/CONTRIBUTING.md#supported-browsers).
- [ ] I have tested this PR in dark mode and light mode (if applicable).

### CODE REVIEW

File: src/components/NftRewards/NftPreview.tsx [hunk 1]
```diff
@@ -45,7 +45,7 @@ export function NftPreview({
           <h3 className="mb-0 uppercase text-slate-100">
             <Trans>{projectMetadata?.name}</Trans>
           </h3>
-          <CloseOutlined className="pl-4" onClick={onClose} />
+          <CloseOutlined className="pl-4 text-slate-100" onClick={onClose} />
```
Comment (johnnyd-eth) [inline src/components/NftRewards/NftPreview.tsx:48]: (unrelated fix to preview close button color)

**Comment by wraeth-eth** [comment]
image
nit: numbers between redemption value and worth dont match (rounded up)

**Review by wraeth-eth — COMMENTED** [review]
back it!
